pkg/config/config.go

```diff
@@ -12,6 +12,7 @@ type Providers struct {
 	Qemu       []Qemu       `yaml:"qemu"`
 	Photon     []Photon     `yaml:"photon"`
 	Xen        []Xen        `yaml:"xen"`
+	Openstack  []Openstack  `yaml:"openstack"`
```

Comment:
I'd leave it like this. Since also `Vsphere` and `Virtualbox` are named this way my guess is that naming convention is "make it lowercase, only capitalise to make attribute public".